modules/openthread/platform/ble.c

```diff
@@ -433,6 +433,30 @@ otError otPlatBleGapAdvSetData(otInstance *aInstance, uint8_t *aAdvertisementDat
 	return OT_ERROR_NONE;
 }
 
+otError otPlatBleGapAdvUpdateData(otInstance *aInstance, uint8_t *aAdvertisementData,
```

Comment:
Isn't usage of `aAdvertisementData` missing?

Reply:
The aAdvertisementData is actually buffer fro the same file take a look at otPlatBleGetAdvertisementBuffer

Reply:
Right, that's what I was missing :)